README.md

```diff
@@ -88,3 +88,16 @@ Make your changes.
 Commit your changes (git commit -m 'Add some feature').
 Push to the branch (git push origin feature/your-feature-name).
 Open a pull request.
+
+---
+
+## Our Awesome Contributors 🚀
+
+- Thank you for your valuable contribution to this project. We truly appreciate your skills and dedication. Keep up the great work!
+- Make sure you show some love by giving ⭐ to our repository.
+
+<div align="left">
+  <a href="https://github.com/AlfiyaSiddique/TastyTrails">
+    <img src="https://contrib.rocks/image?repo=AlfiyaSiddique/TastyTrails&&max=10" />
```

Comment:
what is this like dude..

![image](https://github.com/user-attachments/assets/42ace179-8e28-406f-9f29-a77bf2ab1dae)

i think these are not the contributors from this repo..